doc/quick_start/contributor-guide.rst

```diff
@@ -75,25 +75,52 @@ Here are guidelines for the review process that you and the maintainers will fol
 #. The general commit message format is::
 
      <component>: <summary>
+
      <description>
+
      [Fixes|Related-to] OP-<number>
```

Comment:
While we're at it, can we add a colon in here, i.e.; [Fixes|Related-to]: OP-<number>. That would make the explanation more consistent with the example below,